### PR TITLE
Fix favicon not rendering properly in google search.

### PR DESCRIPTION
**Pull request by jmuzina**
## Done

Updates site to use 48x48 version of the Vanilla logo as a favicon, instead of the old 32x32 version. We are also now serving a favicon locally for search engines, following the practice done by [c.com](https://github.com/canonical/canonical.com/blob/f866a0da17f41e9e02f8a6f6786483ea040f5ce9/templates/base_index.html#L100-L104) & [u.com](https://github.com/canonical/ubuntu.com/blob/f3f3592e5a8d5dc6c1a33f7dc9cbbae2a4059f22/templates/templates/base.html#L52-L53).

See [Google favicon rendering documentation](https://developers.google.com/search/docs/appearance/favicon-in-search) for more information.

Fixes [WD-12464](https://warthogs.atlassian.net/browse/WD-12464), [MM thread](https://chat.canonical.com/canonical/pl/ja9ya3tgkjdpfkhgidt1x8xama)

## QA

- Open [demo](https://vanilla-framework-5183.demos.haus/)
- Verify favicon renders as expected.
- Open [locally served favicon](https://vanilla-framework-5183.demos.haus/static/images/favicons/vanilla_favicon_48px.png)
- Verify it also renders as expected.

## Google Recrawl
After this is merged, it may take a few days for the icon to change on Google search results. If it still hasn't been fixed after some time, we should [submit a recrawl request](https://developers.google.com/search/docs/crawling-indexing/ask-google-to-recrawl) which should fix the issue.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

- [x] PR should have one of the following labels to automatically categorise it in release notes:
  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
- [ ] Vanilla version in `package.json` should be updated relative to the [most recent release](https://github.com/canonical/vanilla-framework/releases/latest), following semver convention:
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)
- [ ] Any changes to component class names (new patterns, variants, removed or added features) should be listed on the [what's new page](https://github.com/canonical/vanilla-framework/blob/main/releases.yml).

[WD-12464]: https://warthogs.atlassian.net/browse/WD-12464?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jmuzina** [comment]
@lyubomir-popov Please provide an export of the new Vanilla logo to use for the favicon, per MM discussion

**Comment by jmuzina** [comment]
@bartaz @pastelcyborg @lyubomir-popov Updated with new logo, please review :)

**Review by bartaz — APPROVED** [review]
LGTM, thanks!

File: templates/_layouts/_root.html [hunk 1]
```diff
@@ -14,7 +14,9 @@
 
     <title>{% block title %}{{ title }}{% endblock %}{% if self.title() %} | {% endif %}Vanilla documentation</title>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <link rel="icon" href="https://assets.ubuntu.com/v1/ab36e6ed-vanilla_favicon_32px.png" type="image/x-icon" />
+    <link rel="icon" type="image/x-icon" sizes="48x48" href="https://assets.ubuntu.com/v1/f1464d22-vanilla_favicon_48px.png"  />
```
Comment (bartaz) [inline templates/_layouts/_root.html:17]: is this correct to have `type="image/x-icon"` when it's a png, not an .ico?

Comment (jmuzina) [reply templates/_layouts/_root.html:17]: @bartaz `image/png` is technically correct - will adjust

**Comment by lyubomir-popov** [comment]
LGTM

**Pull request merged**